shopinvader_base_address_city/README.rst

```diff
@@ -0,0 +1,72 @@
+=============================
+Shopinvader Base Address City
+=============================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:df08260bb029cf91a4dddf225b7ccffeb710a7bc21b57ceb0790ccbc6f400b8a
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-shopinvader%2Fodoo--shopinvader-lightgray.png?logo=github
+    :target: https://github.com/shopinvader/odoo-shopinvader/tree/14.0/shopinvader_list_cities
+    :alt: shopinvader/odoo-shopinvader
+
+
+|badge1| |badge2| |badge3|
+
+REST Services for To list Brazilian cities.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Call this endpoint every time you need to fetch Brazilian cities from a state.
```

Comment:
@cristiano-one I've the feeling that the description is not too restrictive... Are you sure we can only get Branzilian Cities???

Reply:
I adapted it to respond to any country, depending on what is contained in the base.